popup-info-box-web-component/main.js

```diff
@@ -69,9 +70,9 @@ class PopUpInfo extends HTMLElement {
     `;
 
     // Attach the created elements to the shadow dom
-    shadow.appendChild(style);
+    this.shadow.appendChild(style);
```

Comment:
There is no `shadow` property on `this`, I think this should be `shadowRoot` that is defined on `Element`  and initialized via the `attachShadow()` method in the constructor.